cmd/tendermint/commands/init.go

```diff
@@ -32,33 +36,43 @@ func init() {
 }
 
 func initFiles(cmd *cobra.Command, args []string) error {
+	var mode string
```

Comment:
if a user doesnt specify a mode but specified it in the config.toml this should pass right?

Reply:
you meant that they are running `tendermint init` when they already have initialized a config file?